modules/virt-vmware-comparison.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/about_virt/about-virt.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-vmware-comparison_{context}"]
+= Comparing {VirtProductName} to {vmw-full}
+
+If you are familiar with {vmw-first}, the following table lists {VirtProductName} components that you can use to accomplish similar tasks. However, because {VirtProductName} is conceptually different from {vmw-short}, and much of its functionality comes from the underlying {product-title}, {VirtProductName} does not have direct alternatives for all {vmw-short} concepts or components.
+
+.Mapping of {vmw-short} concepts to their closest {VirtProductName} counterparts
+[options="header"]
+[cols="2,2,3"]
+|===
+|{vmw-short} concept |{VirtProductName} |Explanation
+
+|Datastore
+|Persistent volume (PV){nbsp}+ +
+Persistent volume claim (PVC)
+|Stores VM disks. A PV represents existing storage and is attached to a VM through a PVC. When created with the `ReadWriteMany` (RWX) access mode, PVCs can be mounted by multiple VMs simultaneously.
+
+|Dynamic Resource Scheduling (DRS)
+|Pod eviction policy{nbsp}+ +
+Descheduler
+|Provides active resource balancing. A combination of pod eviction policies and a descheduler allows VMs to be live migrated to more appropriate nodes to keep node resource utilization manageable.
+
+|NSX
+|Multus{nbsp}+ +
+OVN-Kubernetes{nbsp}+ +
+Third-party container network interface (CNI) plug-ins
+|Provides an overlay network configuration. There is no direct equivalent for NSX in {VirtProductName}, but you can use the OVN-Kubernetes network provider or install certified third-party CNI plug-ins.
+
+|Storage Policy Based Management (SPBM)
+|Storage class
+|Provides policy-based storage selection. Storage classes represent various storage types and describe storage capabilities, such as quality of service, backup policy, reclaim policy, and whether volume expansion is allowed. A PVC can request a specific storage class to satisfy application requirements.
+
+|vCenter +
+vRealize Operations
```

Comment:
I wonder if bullet points would improve clarity here (and in the vSwitch/DvSwitch cell)